genie-client/src/test/resources/application-integration.yml

```diff
@@ -21,12 +21,13 @@ genie:
     server:
       enabled: false
   jobs:
-    dir:
-      location: "file:///tmp/"
-    archive:
-      location: /archive/location
+    locations:
+      archives: /archive/location
+      jobs: file:///tmp/genie/jobs/
 
 spring:
+  autoconfigure:
+    exclude: org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration, net.devh.springboot.autoconfigure.grpc.server.GrpcServerAutoConfiguration
```

Comment:
Could this be written as YAML array? Just to make it more legible..

Reply:
Yeah it looks like you can: https://github.com/spring-projects/spring-boot/commit/abfd139d8f24f45f8290a4a1fcdd0c649079bfe4

All the examples I'd found were like this so I copied it. I'll fix this in a subsequent commit